tests/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
 enable_testing()
 include_directories(../src)
```

Comment:
This would break compilation for non Windows platforms as above `include_directories` would use win32 dlfn.h file instead of correct (e.g. GNU libc) dlfnc.h file.